tox.ini

```diff
@@ -29,6 +29,7 @@ coverage-config = tox.ini
 [run]
 omit =
   *.jinja2
+  src/aac/ui/*
```

Comment:
I added this since we don't have an established method for testing the UI at the moment.